.travis.yml

```diff
@@ -1,59 +1,17 @@
-sudo: false
 language: ruby
+dist: bionic
 cache: bundler
 rvm:
-  - 2.0
-  - 2.1
-  - 2.2.2
-  - 2.3.0
-  - 2.4.0
-  - 2.5.0
-  - rbx
+  - 2.5.9
+  - 2.6.10
+  - 2.7.6
+  - 3.0.4
+  - 3.1.2
 env:
-  - ACTIVERECORD=3.0.0
-  - ACTIVERECORD=3.1.0
-  - ACTIVERECORD=3.2.0
-  - ACTIVERECORD=4.0.0
-  - ACTIVERECORD=4.1.0
-  - ACTIVERECORD=4.2.0
-  - ACTIVERECORD=5.0.0
-  - ACTIVERECORD=5.1.1
-matrix:
-  exclude:
-    - rvm: 2.0
```

Comment:
Were we seriously testing different versions of RVM as part of the build matrix? :laughing: 

Good catch

Reply:
I think that was just telling Travis to not build that ruby version + activerecord version. In this case they are all super old and we are no longer testing them as suggested in this branch, so I removed them.

Reply:
Oh, so `rvm` here means `ruby`?

Reply:
Yeah per https://docs.travis-ci.com/user/languages/ruby/#specifying-ruby-versions-and-implementations